.github/workflows/release.yaml

```diff
@@ -149,7 +149,7 @@ jobs:
       contents: write
     steps:
       - name: Download artifacts from build
-        uses: actions/download-artifact@d3f86a106a0bac45b974a628896c90dbdf5c8093 # v4.3.0
+        uses: actions/download-artifact@634f93cb2916e3fdff6788551b99b062d0335ce0 # v5.0.0
```

Comment:
We don't download artefacts by IDs, looks like we're unaffected by the breaking changes from v5.